tests/testsuite/doc.rs

```diff
@@ -1879,24 +1879,25 @@ fn doc_message_format() {
 
     p.cargo("doc --message-format=json")
         .with_status(101)
-        .with_json_contains_unordered(
```

Comment:
Does the snapbox patch now correctly handle unordered JSON lines, or this test doesn't need unordered assertion?

Reply:
If by "now correctly" you mean "implicitly", then no, as that would not be correct.  I am not seeing anything in this test case that needs `.unordered()` applied.

Reply:
I meant to comment under `doc_json_artifacts` but that one doesn't seem to need unordered either.